Driver_Mutation_Results/workflow/docker/driver_mutation_results/bin/calculate_driver_results.R

```diff
@@ -97,6 +97,12 @@ parser$add_argument(
   default = "status"
 )
 
+parser$add_argument(
+  "--num_significant_digits",
+  type = "integer",
+  default = 6
```

Comment:
Should the default be to round? I'm wondering if it would be better to not round as a default.

Reply:
That's fair. I've updated the default to not round, and the example JSON config does the rounding for the purposes of comparison.